var/spack/repos/builtin/packages/py-cftime/package.py

```diff
@@ -13,8 +13,17 @@ class PyCftime(PythonPackage):
     homepage = "https://unidata.github.io/cftime/"
     url = "https://github.com/Unidata/cftime/archive/refs/tags/v1.0.3.4rel.tar.gz"
 
+    version("1.6.4", sha256="38970aa0d0ed9ed6b1d90f2cff2301b7299ae62d38e39a540400ab00edb4d2ce")
     version("1.0.3.4", sha256="f261ff8c65ceef4799784cd999b256d608c177d4c90b083553aceec3b6c23fd3")
 
-    depends_on("py-setuptools@18.0:", type="build")
-    depends_on("py-cython@0.19:", type="build")
-    depends_on("py-numpy", type=("build", "run"))
+    with when("@1.0.3.4"):
```

Comment:
Can you keep the structure of dependency listing as in the other packages in this PR, i.e. interspersed instead of block-per-version?